test/data/audio-tracks.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
```

Comment:
Yeah, it seems to work within our tests, but I'm happy to switch to double quoting if we want more compliant JSON.